lib/help-build-dictionary.js

```diff
@@ -60,7 +61,7 @@ module.exports = function helpBuildDictionary (options, cb) {
         }
 
         // Merge module into dictionary
-        _.merge(dictionary, module);
+        mergeDictionaries(dictionary, module);
```

Comment:
👍   This is explicitly designed as a deep merge, used only when the `aggregate` flag is enabled.  Thus we expect Sails' conventional deep-merging strategy for config, which is what we have here.